docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/azure.md

```diff
@@ -5,14 +5,200 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guid
 ---
 
 # Azure
-
 !!!include(developer-docs/latest/setup-deployment-guides/deployment/snippets/deployment-guide-not-updated.md)!!!
 
-This is a step-by-step guide for deploying a Strapi project to [Azure](https://azure.microsoft.com/en-us/). Databases can be on a [Azure Virtual Machine](https://azure.microsoft.com/en-us/services/virtual-machines/), hosted externally as a service, or via the [Azure Managed Databases](https://azure.microsoft.com/en-us/services/postgresql/). Prior to starting this guide, you should have created a [Strapi project](/developer-docs/latest/getting-started/quick-start.md). And have read through the [configuration](/developer-docs/latest/setup-deployment-guides/deployment.md#application-configuration) section.
+This is a step-by-step guide for deploying a Strapi project to [Azure](https://azure.microsoft.com/) using Platform-as-a-Service (PaaS). If you're interested in using Infrastructre-as-a-Service (IaaS) refer to [IaaS deployment guide](#iaas-deployment-guide) further down. Prior to starting this guide, you should have created a [Strapi project](/developer-docs/latest/getting-started/quick-start.md) and have read through the [configuration](/developer-docs/latest/setup-deployment-guides/deployment.md#application-configuration) section.
+
+## Azure Install Requirements
+
+- You must have an [Azure account](https://azure.microsoft.com/free/) before doing these steps.
+
+## PaaS Deployment Guides
+
+There are three ways which you can deploy the Azure resources, via the [Azure Portal](#creating-resources-via-the-azure-portal), via the [Azure CLI](#creating-resources-via-the-azure-cli) or via an [Azure Resource Manager template](#deploy-with-an-azure-resource-manager-template).
+
+When Strapi is running in a PaaS hosting model, a custom storage provider will be required to avoid the transient disk of the PaaS model, [which is covered towards the end](#configure-strapi-for-azure-appservice).
+
+### Required Resources
+
+There are three resources in Azure that are required to run Strapi in a PaaS model, [AppService](https://azure.microsoft.com/services/app-service/?WT.mc_id=javascript-37811-aapowell) to host the Strapi web application, [Storage](https://azure.microsoft.com/product-categories/storage/?WT.mc_id=javascript-37811-aapowell) to store images/uploaded assets, and a database, Azure has managed MySQL and Postgres to choose from (for this tutorial, we'll use MySQL, but the steps are the same for MySQL).
+
+### Creating Resources via the Azure Portal
+
+In this section we'll use the Azure Portal to create the required resources to host Strapi.
+
+1. Navigate to the [Azure Portal](https://portal.azure.com/?WT.mc_id=javascript-37811-aapowell)
+
+1. Click **Create a resource** and search for _Resource group_ from the provided search box
```

Comment:
All of your steps have the same number

Reply:
Numbers for an ordered list auto-increment and don't require explicit stating (unless you need to restart lists due to a break introduced), see the published build here https://documentation-k9ok0crap-strapijs.vercel.app/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/azure.html#creating-resources-via-the-azure-portal and the numbers increment as expected.

By using the same start number, it's easier to introduce new steps, reorder steps, etc. without bulk changes throughout the doc.

The original doc (on deploying to a VM) didn't use ordered lists, it uses numbered headings (which I find breaks the anchor links), which is why they were explicitly stated.

Reading the [writing guide](https://handbook.strapi.io/user-success-manual/strapi-documentation-style-guide#1d4a11f683d047cd9128664823db2b7d) it didn't seem to state one way or the other, but if the intent is to have the Markdown explicitly reflect the steps, I'll update to reflect.

Reply:
I agree it works with our VuePress setup but it still makes things easier to review/debug/discuss with colleagues. So when you have time, it would be nice if you could please update these numbers.